### PR TITLE
customize docker plugin

### PR DESCRIPTION
**Pull request by grafjo**
so we can push to a different registry - e.g. a private one.

Publish Amphora docker iamge to a private registry:

```bash
export AMPHORA_IMAGE_TAG=latest
mvn package dockerfile:push \
 -Ddocker.repository=registry.example.org/carbynestack/amphora-service \
 -Ddocker.tag=$AMPHORA_IMAGE_TAG \
 -Ddockerfile.skip=false
```

### CODE REVIEW

**Review by strieflin — CHANGES_REQUESTED** [review]
The changes in formatting (new lines in imports and reformatting of comment) are artifacts of not using `pre-commit`. Please adhere to the [developer guide](https://github.com/carbynestack/carbynestack/blob/master/DEVELOPING.md) for instructions how to set it up correctly. This is a prerequisite for any PR to be accepted (FYI: @sbckr).

**Comment by grafjo** [comment]
> The changes in formatting (new lines in imports and reformatting of comment) are artifacts of not using `pre-commit`. Please adhere to the [developer guide](https://github.com/carbynestack/carbynestack/blob/master/DEVELOPING.md) for instructions how to set it up correctly. This is a prerequisite for any PR to be accepted (FYI: @sbckr).

thx for the feedback - solved those issues!

**Review by strieflin — CHANGES_REQUESTED** [review]

File: amphora-service/pom.xml [hunk 1]
```diff
@@ -18,6 +18,9 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
     <properties>
+        <docker.repository>ghcr.io/carbynestack/${project.artifactId}</docker.repository>
```
[the remark this answers is not in the document]
Comment (grafjo) [reply amphora-service/pom.xml]: requested change is done

**Review by strieflin — APPROVED** [review]
LGTM

**Pull request merged**